### ISSUE TITLE
Push to Prod Enhancement

### ISSUE THREAD

**Issue by mlucrida3980** [Arrowhead-Credit-Union/Card-Compromise#3]
Some of the code needs to be hard coded every time there is a push to production or a pull to development.  We need an enhancement that allows us to configure the runtime settings to either use production or development resources.  

No official design has been suggested, so we are open to any.

**Comment by acu-barchibald** [comment]
That's a good point Matt.  We need some sort of base file/folder structure for each type of project (web, script, db, etc) and more to the point, a prod/test configuration file that can be changed with the environment.  

There are many ways to accomplish this, for instance, we can create 2 config files for each project (dev/prod) and in the code, do a simple conditional statement that can detect if you're on a pc or a server and choose the corresponding config file automatically.  

Any thoughts?

**Comment by mlucrida3980** [comment]
Yeah, I've tried a couple versions of implementing config files.  I think the fastest way to implement it would be to just make a main production config file separate from the main source file.  Each of us could then create our own local copies to point to which ever directories or settings we need.  

As long as we push to our own feature branches we can be sure that we don't overwrite the production version when we do pull requests.

**Comment by acu-barchibald** [comment]
There must be a clever way we can dynamically detect if the script/process is in production or not, just in case one of us forgets to update the config file.  Maybe we can do a check for the name of the machine it's running from, and if it starts with "PC" or "LT" we know it should be going to the dev environment....

**Comment by mlucrida3980** [comment]
Technically, python's best practice would be to create projects as a package.  Then in the __init__.py file we can do the checks there.  That would also give us an easier option to run tests on any of our projects.  I think I already have some alpha versions of a python framework sitting around.  We could use that as a starting point.
